### ISSUE TITLE
blacklist

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#936]
Nobody seems to be watching blacklist objection emails when DLM is out. There are lots, including from traveling UAM Curators, someone with a unm.edu email, etc.

**Comment by ccicero** [comment]
This is one of the many reasons why we need programmer redundancy. We can't expect Arctos users to be doing this sort of thing.

**Comment by dustymc** [comment]
Perhaps we could start with the goal of reliably paying one person, then build redundancy from there?

This is not really threatening - it cannot lead to data loss etc. - but I'm sure it's very frustrating for locked-out users. And I'm fairly certain I can safely automate some more stuff related to this particular problem. 

Other things are potentially threatening and do need a human of some sort at least occasionally involved; SOMEONE needs to be regularly looking at logs. Emailing the network folks is almost always sufficient to protect data, which is better than nothing. That situation definitely deserves more discussion - file a new Issue (or schedule it in the next SC/AWG/?? meeting??).

**Comment by dustymc** [comment]
- intrusion reports are more widely distributed (based on severity), have better documentation
- subnets are auto-blacklisted after 10 IPs are blacklisted
  - reduces application variable load
  - better scary-subnet alerts
  - possible increased chance of catching problems before they happen
- better history - can now track past blocks from the Arctos forms
- better reporting - blacklist IP form rebuilt to show historical data
- better integration - tighter link between blocked IPs and subnets
- more self-service; users can unblock auto-blocked subnets
- more precise control 
  - remains possible to hard-block subnet
  - possible to add finer-grained IP blocking (eg, could implement hard IP blocks)
- more modular - forms and functions have been consolidated
